src/Refinery/String/MakeClickable.php

```diff
@@ -38,14 +40,15 @@ public function transform($from) : string
         $endOfMatch = 0;
         $stringParts = [];
         $matches = [];
-        while (1 === preg_match('@(^|[^[:alnum:]])(((https?://)|(www.))[^[:cntrl:][:space:]<>\'"]+)([^[:alnum:]]|$)@', substr($from, $endOfMatch), $matches)) {
+        
+        while (mb_ereg(self::PATTERN, substr($from, $endOfMatch), $matches)) {
```

Comment:
@mjansenDatabay and @lscharmer I has to replace preg_match with mb_ereg since all tests with multibytes failed locally on my machine, looked like this:

![image](https://user-images.githubusercontent.com/6661332/165890218-a7b12f30-1d8a-4ec9-b744-8e7f79943ef0.png)

Reply:
It's what @Amstutz mentioned in #4441

Reply:
Thx @chfsx